client/state/comments/constants.js

```diff
@@ -6,6 +6,13 @@ export const PLACEHOLDER_STATE = {
 	ERROR: 'ERROR',
 };
 
+// values conveniently also correspond to css classNames to apply
+export const POST_COMMENT_DISPLAY_TYPES = {
```

Comment:
I think to match the way we've set up constants elsewhere in Calypso, there should b e one for each display type, e.g.

`export const POST_COMMENT_DISPLAY_TYPE_SINGLE_LINE = 'is-single-line';`

Reply:
I know it'll make `isValidExpansionsAction` a bit clunkier, but it feels right.

Reply:
the part I don't like about the clunkiness of isValidExpansion is that if we add more display types we need to make changes in multiple places whereas with this object-for-types method things will just work